logger.go

```diff
@@ -18,7 +18,9 @@ func NewPlanetScaleLogger() *zap.Logger {
 }
 
 // New creates a new default PlanetScale Logger.
-var New = NewPlanetScaleLogger
+func New() *zap.Logger {
```

Comment:
Turns out, these are slightly better as actual function calls even though they are identical. It helps godoc out so it's an actual function, and not a variable.